tools/tool.bzl

```diff
@@ -0,0 +1,38 @@
+"""Makes tool macros for terraform et al"""
```

Comment:
can delete this file?

Reply:
I copied this approach for invoking terraform from silo.